app/views/admin/role_appointments/edit.html.erb

```diff
@@ -1,40 +1,74 @@
-<% page_title "Edit appointment to #{@role_appointment.role.name}"  %>
+<% content_for :page_title, "Edit appointment to #{@role_appointment.role.name}"  %>
+<% content_for :title, "Edit appointment to #{@role_appointment.role.name}" %>
+<% content_for :title_margin_bottom, 6 %>
+<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @role_appointment, parent_class: "role_appointment")) %>
 
-<div class="row">
-  <div class="col-md-8">
-    <h1>Edit appointment to position of <%= @role_appointment.role.name %> </h1>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_for [:admin, @role_appointment] do |form| %>
+      <%= render 'admin/role_appointments/form', form: form %>
 
-    <%= form_for [:admin, @role_appointment], html: {class: 'well'} do |form| %>
-      <%= render partial: 'admin/role_appointments/form', locals: { form: form } %>
-
-      <%= form.save_or_cancel cancel: edit_admin_role_path(@role_appointment.role) %>
+      <div class="govuk-button-group">
+        <%= render "govuk_publishing_components/components/button", {
```

Comment:
I think we can make this a bit simpler

```
  <div class="govuk-button-group">
        <%= render "govuk_publishing_components/components/button", {
          text: "Save",
          data_attributes: {
            module: "gem-track-click",
            "track-category": "form-button",
            "track-action": "role-appointment-button",
            "track-label": "Save"
          }
        } %>
```

If you look a the [component](https://components.publishing.service.gov.uk/component-guide/button), all you need to provide is text and it defaults to having a type of submit etc.

Providing a name/value is only required when the controller is figuring out where to send a user submits a form when there are 2 save buttons. So it can do something like

```
if object.save 
  if params[:save] == "save and continue"
    redirect to x
  else
    redirect to x
  end
else 
  render same page with errors
end
```

Reply:
Cool, yes, that makes perfect sense as well. That was me being lazy and copy pasting over from the role edit page, without thinking hard about whether each of those parameters are actually required.

Reply:
No worries happens to me all  the time